src/main/java/core/basesyntax/figures/suppliers/RoundCalculator.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.figures.suppliers;
+
+import core.basesyntax.figures.Figure;
+
+public class RoundCalculator {
+    public double roundCalc(double number) {
+        int nearestTenth = Figure.NEAREST_TENTH;
+        return (double) Math.round(number * nearestTenth) / nearestTenth;
+    }
+}
```

Comment:
if you'd like to format the output of draw (how many digits after dot), you can use String.format instead